### DISCUSSION TITLE
Feature request

### DISCUSSION THREAD

**Discussion by DevVigilante** [Sketchware-Pro/Sketchware-Pro#730] [Ideas]
👉Feature request : In Inject property this is hard to code without an code editor please add code editor option in inject property of wedget

👉Feature request : Please add a option which allow us to create an custom wedget which have three properties
> Inject Property
> Can place view inside that created wedget(As shown in below image)
> Width height ,Orientation(Just only for editor and not for implement in source code)

![Screenshot_2023-01-29-14-18-39-655_com sketchware remod](https://user-images.githubusercontent.com/96680248/215315636-cb009502-0d18-47fc-84f8-f6d526d0fef5.jpg)

If you can't do that ignore it for further but please add code editor to inject property

**Comment by j-px** [comment]
2nd one is a duplicate of #471.

**Reply by DevVigilante** [reply]
> 2nd one is a duplicate of #471.

Ok but 1st should be implemented 😥
